openoereb/pyramid_oereb
# Adapt Topic Page

closes #1215
TOTEST:
- use the mfp pr https://github.com/openoereb/pyramid_oereb_mfp/pull/80
- turn off pdfA. For instructions on this see https://github.com/openoereb/pyramid_oereb_mfp/pull/77

TODO:
- [x] fix test_get_sorted_hints
- [x] fix test_get_sorted_law 
- [x] fix mfp extract test
